radixdlt/src/main/java/com/radixdlt/consensus/VerifiedLedgerHeaderAndProof.java

```diff
@@ -88,8 +90,8 @@ public VerifiedLedgerHeaderAndProof(
 		this.signatures = Objects.requireNonNull(signatures);
 	}
 
-	public static VerifiedLedgerHeaderAndProof genesis(Hash accumulator) {
-		LedgerHeader genesisLedgerHeader = LedgerHeader.genesis(accumulator);
+	public static VerifiedLedgerHeaderAndProof genesis(Hash accumulator, BFTValidatorSet nextValidators) {
+		LedgerHeader genesisLedgerHeader = LedgerHeader.genesis(accumulator, nextValidators);
```

Comment:
Same suggestion as for LedgerHeader.create() - add version of this method without last parameter.